charts/datafuse/templates/hpa.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
```

Comment:
In alpha stage, it is fine, I am worrying about its behavior may incur flaky issues in production environment, maybe we could consider to use keda.sh in future release